tests/core/chain-object/conftest.py

```diff
@@ -7,3 +11,14 @@
     chain as valid_chain,
     chain_without_block_validation as chain,
 )
+from eth_keys import KeyAPI
+
+
+@pytest.fixture
+def unfunded_address():
```

Comment:
Assuming this is the address of the private key from `unfunded_address_private_key` how about changing this to return `unfunded_address_private_key.public_key.to_canonical_address()`